src/components/DVDsItemsandlists/DVDsItem.js

```diff
@@ -0,0 +1,25 @@
+import classes from './DVDsItem.css';
+
+function DVDsItem(props){
+    return (
+    <div>
+    <ul>
+    <li>        
```

Comment:
This shouldn't be a list item, and it should not be in an unordered list, as it is only a single item